### ISSUE TITLE
Does rxjs-no-wholesale still had a life meaning ?

### ISSUE THREAD

**Issue by evanliomain** [cartant/rxjs-tslint-rules#36]
According the [rxjs v6 breaking changes](https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#breaking-changes-1), a lot of features will no longer be available with a deep import.

So, we will have to import interface like this: 
```typescript
import { Observable, Subject } from 'rxjs';
```

Instead of:
```typescript
import { Observable } from 'rxjs/Observable';
import { Subject } from 'rxjs/Subject';
```

As the `rxjs-no-wholesale` rule suggest.

So, does we still need this rule for the next rxjs release ?

**Comment by cartant** [comment]
No, it won't be needed, but I don't anticipate making any changes to the rules until there is a version 6 beta. When there is a beta release, I'll publish a version of this package using the `beta` tag, etc.

**Comment by DaSchTour** [comment]
You can simply disable this rule if you use rxjs 6. I don't think this needs any changes in the rules but just in their usage.

**Comment by evanliomain** [comment]
@DaSchTour That what I do, even if I still use rxjs 5.
However, for the futur (ie: rxjs v6), this rules will be in opposition with the library. So it will be pointless and misunderstood to keep it.

**Comment by DaSchTour** [comment]
@evanliomain So this may be a point that should be documented. Just keep in mind that not every project will switch to rxjs 5 immediately after release. So it might make sense to keep this rule for years after release of rxjs 6 and usage of rules with v5 and v6 needs to be documented. I bet there will be new rules that will just make sense with rxjs 6.

**Comment by cartant** [comment]
FYI, I've just published `4.0.0-rc.0` under the `rc` tag:

```
$ npm dist-tag ls rxjs-tslint-rules
latest: 3.16.0
rc: 4.0.0-rc.0
```

It supports both RxJS version 5 and 6. If a version 6 RxJS peer is found, the import-related rules are deprecated, as they are no longer required. Other rules should work as expected.

The project now contains fixtures that test the rules with versions 5 and 6 of RxJS.

**Comment by cartant** [comment]
Version 4.0.0 has been published and it supports RxJS versions 5 and 6 and supports `rxjs-compat` being used with version 6.
